flipperdevices/flipperzero-firmware
# Archive: Fix item focus after aborting the Delete operation

# What's new

This PR fixes a bug in the Browser where cancelling a Delete operation made the Browser focus on the first item on the list, with the Pin/Rename/Delete context menu still open. This PR fixes the focus issue, and hides the context menu when Delete is selected, so the behaviour is consistent with Rename where it was happening already.

# Verification 

1. Open Browser and navigate anywhere where there are multiple items on the screen. In my test, it was `apps`\Games.
2. Focus on any entry that is **not** the first.
3. Select the entry and choose `Delete`, then `Cancel`.
4. Observe that:
   * Without this PR, the context menu is still open, and the menu is focused on the top entry.
   * With this PR, the context menu is not open, and the menu is focused on the previously selected entry.

I have verified that deleting items does not break the flow and the menu remains usable (just focuses on the entry that used to be next to the deleted one).

# Checklist (For Reviewer)
